docs/experiments-nfcorpus.md

```diff
@@ -26,10 +26,10 @@ How to actually train such models will be covered later.
 
```

Comment:
Something weird happened here? Please revert and only add in the repro log entry?